src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/IssueFurtherEvidenceHandler.java

```diff
@@ -105,27 +125,24 @@ private void issueEvidencePerDocumentType(SscsCaseData caseData, List<FurtherEvi
         log.info("Issued for caseId {}", caseData.getCcdCaseId());
     }
 
-    private void postIssueFurtherEvidenceTasks(SscsCaseData caseData) {
-        log.debug("Post Issue Tasks for caseId {}", caseData.getCcdCaseId());
+    private void postIssueFurtherEvidenceTasks(long caseId, IdamTokens idamTokens) {
+        log.debug("Post Issue Tasks for caseId {}", caseId);
         try {
-            if (caseData.getReasonableAdjustmentsLetters() != null) {
```

Comment:
Is this if statement lost / no longer needed. Happy to have a call if I have missed something @nitinprabhuhmcts

Reply:
Spoke with @nitinprabhuhmcts and resolved